src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,21 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePage {
+    private WebDriver driver;
+    private By formLogin = By.cssSelector("a[href*='showlogin']");
```

Comment:
This is not a good locator, with the slides presented last Friday try to find a new locator. After seeing the issue that you explained you can fix it in ha better way.